core/src/main/java/com/tweet/core/model/Tweet.java

```diff
@@ -0,0 +1,11 @@
+package com.tweet.core.model;
+
+import javax.persistence.Entity;
+
+@Entity
+public class Tweet {
+
+    private String username;
+    private String content;
+    private Long date;
```

Comment:
Why long? wanna parse it all the time? :) LocalDate would be better, also date may be not to descriptive.
We can leave date as name, but in this case please add:
@Column(name = "CREATED_DATE")

Reply:
yet it would be better to change date to createdDate - this was hibernate will name column CREATED_DATE

Reply:
Since Java 8, the new Java Date and Time API is available for dealing with temporal values. This API fixes many of the problems of java.util.Date and java.util.Calendar classes.

The types from the java.time package are directly mapped to corresponding SQL types. So there’s no need to explicitly specify @Temporal annotation:

    LocalDate is mapped to DATE
    LocalTime and OffsetTime are mapped to TIME
    Instant, LocalDateTime, OffsetDateTime and ZonedDateTime are mapped to TIMESTAMP